src/NETCore.MailKit/Core/EmailService.cs

```diff
@@ -219,8 +271,29 @@ private void SendEmail(string mailTo, string mailCc, string mailBcc, string subj
             //set email encoding
             body.SetText(encoding, message);
 
+            //add multipart
+            Multipart multipartBody = new Multipart("mixed")
+            {
+                body
+            };
+
+            // add attachments
+            if (attachments != null)
+                foreach (var attach in attachments)
+                {
+                    var mimeType = MimeTypes.GetMimeType(attach).Split('/');
+                    multipartBody.Add(new MimePart(mimeType[0], mimeType[1])
+                    {
+                        IsAttachment = true,
+                        Content = new MimeContent(File.OpenRead(attach), ContentEncoding.Default),
```

Comment:
Can we open files without locking them?

`File.Open(attach, FileMode.Open, FileAccess.Read, FileShare.ReadWrite | FileShare.Delete)`